game_session.js

```diff
@@ -63,4 +66,48 @@ module.exports = class GameSession {
     getDefaultBeginningCutoffYear() {
         return BEGINNING_SEARCH_YEAR;
     }
+
+    resetGender() {
```

Comment:
Do we need to have a reset function for each individual parameter? Why not just reset everything at once.

Reply:
If user enters an invalid input, just ignore it and keep the old values

Reply:
At this point I don't see the value in having a function to reset all options at once, but it might be valuable in the future (something like `!reset` maybe?). I've created reset functions to prevent code duplication (and for future-proofing if we have to do more advanced tasks in order to reset an option). Then, if we need to create a function to reset everything at once in the future, we can just call all of these helper functions and keep everything modular.

That being said, I can definitely implement keeping old values in the case of invalid input. Are you fine with me making a temporary gender array to store the old values so we have something to reset it to?

Reply:
Yeah I intended it for `!reset`, I just don't see any reason to auto-reset a parameter if an invalid input is entered.   

It'd probably be better to create a temporary array to store the new values, then assigning it to the actual gender array afterwards (assuming no invalid input)